File: 프로그래머스/2/12939. 최댓값과 최솟값/최댓값과 최솟값.java
class Solution {
    public String solution(String s) {
        String[]  str = s.split(" ");
        int[] numbers = new int[str.length];
        int max = Integer.parseInt(str[0]);
        int min = Integer.parseInt(str[0]);
        for(int i = 0;i < str.length;i++){
            numbers[i] = Integer.parseInt(str[i]);
            if(numbers[i]>max){
                max = numbers[i];
            }
            if(numbers[i]<min){
                min = numbers[i];
            }
        }
        return Integer.toString(min)+" "+Integer.toString(max);
    }
}

File: 프로그래머스/2/12939. 최댓값과 최솟값/최댓값과 최솟값.js
function solution(s) {
    let arr = s.split(" ").map((i)=>Number(i));
    let max = arr.sort((a,b)=> b-a)[0];
    let min = arr.sort((a,b)=> b-a)[arr.length -1];
    return `${min} ${max}`
}